packages/@okta/i18n/src/properties/login.properties

```diff
@@ -708,6 +708,8 @@ oie.verify.button = Verify
 
 ## OIE Password
 oie.password = Password
+oie.challenge.password.title = Sign in using your password
```

Comment:
`nit:` can we change the key to `oie.password.challenge.title` ref: https://github.com/okta/okta-signin-widget/pull/1240